### PR TITLE
add git secrets ci ros1

### PR DESCRIPTION
**Pull request by ojasjoshi**
*Issue #, if available:*

*Description of changes:*
Adding git secrets scan action to ROS1 workflow

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by ojasjoshi** [comment]
Blocked by https://github.com/aws-robotics/aws-robomaker-github-actions/pull/17.

Before merging, requires version bump (to 2.3.4) for all aws-robomaker-gh-actions after is merged.

**Review by mm318 — APPROVED** [review]

**Pull request merged**
